conjure_oxide/tests/integration/basic/product/01-simple/input-expected-rule-trace-human.txt

```diff
@@ -0,0 +1,12 @@
+Product([Product([x, y]), z]), 
+   ~~> normalise_associative_commutative ([("Base", 8900)]) 
+Product([x, y, z]) 
+
+--
+
+(Product([x, y, z]) < 15), 
+   ~~> lt_to_ineq ([("Minion", 4100)]) 
+Ineq(Product([x, y, z]), 15, -1) 
```

Comment:
is ineq not the minion one? in which case, should it not only carry atoms instead of full expressions.

Reply:
Agreed, just haven't done this change for all the Minion constraints yet. I'll finish this refactor before marking this PR ready again

Reply:
#566